Tests/EmbeddedScripts.JS.ChakraCore.Tests/ChakraCoreRunnerTests.cs

```diff
@@ -164,11 +172,63 @@ public async void CodeThrowsAnException_SameExceptionIsThrowingFromRunner()
             var exceptionMessage = "Exception from user code";
             var code = $"throw Error('{exceptionMessage}');";
 
-            var runner = new ChakraCoreRunner();
+            using var runner = new ChakraCoreRunner();
             var exception = await Assert.ThrowsAsync<ScriptRuntimeErrorException>(() =>
                 runner.RunAsync(code));
 
             Assert.Equal(exceptionMessage, exception.Message);
         }
+        
+        [Fact]
+        public async Task RegisteringNewGlobalVarBetweenRuns_Success()
```

Comment:
Why do we need separate test sets for each runner? Theoretically they should share the same test cases to ensure runners are interchangeable for the same language (eg javascript)...

Reply:
I will refactor tests in Refactoring branch (check PR #13)